ios-files/LiveActivityView.swift

```diff
@@ -59,6 +59,12 @@ import WidgetKit
             .tint(progressViewTint)
             .modifier(ConditionalForegroundViewModifier(color: attributes.progressViewLabelColor))
         }
+
+        if let progress = contentState.progress {
```

Comment:
I think it would be better not to allow for two progress bars at the same time. We could add `else if let progress` instead, so that this progress bar will be visible only if there is no date.